arlas-hits-exporter/export_data.bash

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+set -o errexit -o pipefail
```

Comment:
It would be good to strengthen the script by adding

```
-o nounset
```

It prevents use of undeclared environment variables. You can then use `if [[ -v VARIABLE_NAME ]]` to check if a variable is defined.